packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -293,10 +294,6 @@ async def start(self) -> None:
 
         :return: None
         """
-        which = shutil.which("go")
-        if which is None:
-            raise Exception("Libp2p Go should me installed")
```

Comment:
why has this been removed?

Reply:
because connection already does the check (using `_check_go_installed()`)